README.md

```diff
@@ -30,28 +30,28 @@ The changes from the main branch include:
 * Corrected unique_id for BMS Cell mV low
 * Added automation to upload generation and consumption data to pvoutput.org
 
-Access to your inverter data can be acheived by connecting the inverter's RS485 modbus to an RS485 to USB adapter or RS485 to WIFI/LAN adapter. Note: this requires basic electronics competencies to connect 2 wires to the inverters CT / COM connector.
+Access to your inverter data can be achieved by connecting the inverter's RS485 Modbus to an RS485 to USB adapter or RS485 to WiFi/LAN adapter. Note: this requires basic electronics competencies to connect 2 wires to the inverter's CT / COM connector.
 
 ---
-** Please note: April 2023, @nathanmarlor has mirrored and extended functionality in this alternative https://github.com/nathanmarlor/foxess_modbus - you may wish to look at that as well!
+** Please note: April 2023, @nathanmarlor has mirrored and extended functionality in this alternative https://github.com/nathanmarlor/foxess_Modbus - you may wish to look at that as well!
```

Comment:
don't think you can change the name of a file as they are case sensitive

Reply:
Oh! You're so right... Might even work (in links at least) but that's not the point, I shouldn't have done a copy and replace "blindly". Will fix